src/coreComponents/physicsSolvers/NonlinearSolverParameters.hpp

```diff
@@ -151,6 +154,15 @@ class NonlinearSolverParameters : public dataRepository::Group
     Parabolic, ///< use parabolic interpolation to define line search scaling factor.
   };
 
+  /**
+   * @brief Coupling type.
+   */
+  enum class CouplingType : integer
+  {
+    FIM,      ///< Fully-implicit coupling
```

Comment:
Just a stylistic suggestion, but maybe write `FullyImplicit` for readability?

Reply:
sure, I don't see why not.